skimage/draw/draw3d.py

```diff
@@ -62,18 +63,19 @@ def ellipsoid(a, b, c, spacing=(1.0, 1.0, 1.0), levelset=False):
 
 
 def ellipsoid_stats(a, b, c):
-    """
-    Calculates analytical surface area and volume for ellipsoid with
-    semimajor axes aligned with grid dimensions of specified `spacing`.
+    """Calculate analytical volume and surface area of an ellipsoid.
+
+    This function makes use of :py:func:`scipy.special.elliprg` to calculate
```

Comment:
I'd remove this line, it's an implementation detail.

Reply:
It is; you can remove it (this addition is commit 321b6011e70b5d60494205ab571fc62ce9d9c727).

But that's how I would address https://github.com/scikit-image/scikit-image/issues/7472#issuecomment-2237639870. 'Everyone knows' that the analytical formula for the volume of an ellipsoid is `4/3*pi*a*b*c`; but it's not clear/guessable a priori which formula will be used for the surface area, so this extra line helps the interested user know where to look and/or, at a glance, know what 'kind' of formula is being used (I mean even really vaguely, not in the sense of 'first kind, second kind, ...').

Reply:
What do you think of 3c7d1607eb17864b18133bcca1b2247078967cbc?

Reply:
In that case, I would write something like:

```
The surface of the ellipsoid is given by

$S=3VR_G\left(a^{-2},b^{-2},c^{-2}\right)$

where $R_G$ is one of the Carlson symmetric forms of the elliptical integrals, implemented as :py:func:`scipy.special.elliprg`.
```

Reply:
Maybe with a reference to the Wikipedia section: https://en.m.wikipedia.org/wiki/Ellipsoid#Surface_area

Reply:
> What do you think of [3c7d160](https://github.com/scikit-image/scikit-image/commit/3c7d1607eb17864b18133bcca1b2247078967cbc)?

LGTM!

Reply:
> In that case, I would write something like:
> 
> ```
> The surface of the ellipsoid is given by
> 
> $S=3VR_G\left(a^{-2},b^{-2},c^{-2}\right)$
> 
> where $R_G$ is one of the Carlson symmetric forms of the elliptical integrals, implemented as :py:func:`scipy.special.elliprg`.
> ```

@pamitabh as a specimen of interested user, what do you say? :wink:

Reply:
Thanks for asking! sounds good, additional comment at #7472.